packages/rum-core/src/domain/getSelectorFromElement.ts

```diff
@@ -54,6 +54,10 @@ export function getSelectorFromElement(targetElement: Element, actionNameAttribu
       return globallyUniqueSelector
     }
 
+    if (isExperimentalFeatureEnabled(ExperimentalFeature.DETACHED_ELEMENT_SELECTOR) && !element.parentElement) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ I don't know if only detached element can have no parent. Maybe it could worth collecting [element.isConnected](https://developer.mozilla.org/en-US/docs/Web/API/Node/isConnected) to check. Wdyt?

Reply:
ok, let's only collect some log first